partner_firstname/__init__.py

```diff
@@ -15,8 +17,5 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
-from . import partner
-from . import res_user
+from . import models
```

Comment:
Git doesn't track moves the way bzr does. It makes no difference. Git tries to match the correct files. It isn't always right.